src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -107,7 +110,7 @@ void should_attach_exception_record_to_the_existing_case_with_evidence_documents
         CaseDetails exceptionRecord = createExceptionRecord("envelopes/supplementary-evidence-envelope.json");
 
         // when
-        invokeCallbackEndpointForLinkingDocsToCase(caseDetails, exceptionRecord);
+        invokeCallbackEndpoint(caseDetails, exceptionRecord, null);
```

Comment:
what does null `searchCaseReferenceType` mean?

Reply:
It means no searchCaseReferenceType is provided. I'll add a comment.